### PR TITLE
Export activation arguments to environment during initialization.

### PR DESCRIPTION
**Pull request by rabbah**
This is part of the work related to https://github.com/apache/incubator-openwhisk/issues/4558 and depends on https://github.com/apache/incubator-openwhisk/pulls/4559.

### CODE REVIEW

File: core/nodejsActionBase/src/service.js [hunk 1]
```diff
@@ -160,6 +160,17 @@ function NodeActionService(config) {
     };
 
     function doInit(message) {
+        if (message.env && typeof message.env == 'object') {
+            Object.keys(message.env).forEach(k => {
```
Comment (chetanmeh) [inline core/nodejsActionBase/src/service.js]: Could have used [Object.entries][1] but looks like thats only from Nodejs 7+. So used approach is more compatabile

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Comment (rabbah) [reply core/nodejsActionBase/src/service.js]: if/when we drop node 6, can switch it.

**Comment by chetanmeh** [comment]
Relevant test also now pass 

```runtime.actionContainers.NodeJs12ActionContainerTests > runtime proxy should export environment variables before initialization PASSED
```

**Comment by rabbah** [comment]
oy you're too fast - I was about to rename the test method name :)

**Comment by chetanmeh** [comment]
Ah you mean change `testEnvPartition`? That would require change in core repo also?

**Pull request merged**